### ISSUE TITLE
Crash due to particle position

### ISSUE THREAD

**Issue by chrisk314** [chrisk314/mesh-sphere-packing#9]
Program crashes for some input particle positions. See below stack trace.

```txt
(venv) csk13@aquila: mesh_sphere_packing $ python build.py -cx 1. 1. 1. -r 0.5 -l 2. 2. 2.
 
/home/csk13/src/MeshSpherePacking/mesh_sphere_packing/splitsphere.py:81: RuntimeWarning: invalid value encountered in arcsin
  phi_1 = np.arcsin(-z * r_inv)
/home/csk13/src/MeshSpherePacking/mesh_sphere_packing/splitsphere.py:82: RuntimeWarning: invalid value encountered in arcsin
  phi_2 = 0.5 * np.pi + np.arcsin(y * r_inv)
Traceback (most recent call last):
  File "build.py", line 43, in <module>
    args.func(args)
  File "build.py", line 10, in build
    segments = splitsphere(args)
  File "/home/csk13/src/MeshSpherePacking/mesh_sphere_packing/splitsphere.py", line 252, in splitsphere
    points, x, y, z, r, Lx, Ly, Lz
  File "/home/csk13/src/MeshSpherePacking/mesh_sphere_packing/splitsphere.py", line 83, in split_sphere_points_segments
    curve_points = get_add_phi_points(phi_1, phi_2, r, ds, fn_x, fn_y, fn_z)
  File "/home/csk13/src/MeshSpherePacking/mesh_sphere_packing/splitsphere.py", line 61, in get_add_phi_points
    n_add = int(np.ceil(phi_disp * r / ds))
ValueError: cannot convert float NaN to integer
```

**Comment by chrisk314** [comment]
Made some quick and dirty fixes (13715cb) and fixed a related bug (9284d81) to get this working for fully internal particles so that work can proceed on #3. Will clean these things up soon when addressing #4.

**Comment by chrisk314** [comment]
Closing for now. Will reopen if new issues are found with different particle positions.
